acceptance/tests/vault/helpers.go

```diff
@@ -85,7 +80,7 @@ func generateGossipSecret() (string, error) {
 func configureGossipVaultSecret(t *testing.T, vaultClient *vapi.Client) string {
 	// Create the Vault Policy for the gossip key.
 	logger.Log(t, "Creating gossip policy")
-	err := vaultClient.Sys().PutPolicy("consul-gossip", gossipPolicy)
+	err := vaultClient.Sys().PutPolicy("gossip", gossipPolicy)
```

Comment:
I've removed the `consul-` prefix from all vault roles to make it a bit less verbose.